.github/workflows/mlc_config.json

```diff
@@ -15,5 +15,6 @@
     {
       "pattern": "^http://proxyserver"
     }
-  ]
+  ],
+  "aliveStatusCodes": [200, 403]
```

Comment:
<img width="995" alt="image" src="https://user-images.githubusercontent.com/4465723/154611309-84d25820-6826-4057-9410-1f2ca7b1ae47.png">
I think it is because of the website policy